cirq/google/op_deserializer.py

```diff
@@ -12,10 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from typing import (
+    Any,
+    Callable,
+    Dict,
+    Optional,
+    Sequence,
+    TYPE_CHECKING,
```

Comment:
nit: remove comma to get formatting that doesn't fill up my screen when I open the file :)

Reply:
Acknowledged. I prefer the vertical list style to the block style, as it makes git diffs cleaner.

Reply:
I think the white space is a big waste.  Plus no one else is doing this in the code base.